x/gov/legacy/v040/migrate_test.go

```diff
@@ -176,8 +176,7 @@ func TestMigrate(t *testing.T) {
 					"height": "123",
 					"info": "foo_upgrade_info",
 					"name": "foo_upgrade_name",
-					"time": "0001-01-01T00:00:00Z",
-					"upgraded_client_state": null
```

Comment:
It seems "ok" to me to be modifying the `.String` of the Plan, but I can add the upgraded client state back as null if people think that'd be better